source/utils/types.ts

```diff
@@ -173,7 +174,7 @@ export interface Options extends PartialDeep<DefaultOptions>, URLOptions, Except
 	useElectronNet?: boolean;
 	form?: Record<string, any>;
 	json?: Record<string, any>;
-	context?: {[key: string]: unknown};
+	context?: Record<string, any>;
```

Comment:
`{[key: string]: any}`

Reply:
Do we need to change form/json too?

Reply:
Yes. `{[key: string]: any}` has special behavior in TypeScript. https://github.com/sindresorhus/typescript-definition-style-guide/pull/8/files `Record` is fine as a input value where the value is not `any`/`unknown` or a return value. For this reason, I just prefer the `{[key: string]: any}` syntax as I can just consistently use it. Yet another thing I dislike about TypeScript. So much inconsistent and hidden behavior.

Reply:
Fixed accordingly 👍